multi_partner_learning.py

```diff
@@ -171,9 +170,9 @@ def compute_test_score(self):
                 model_to_evaluate = sequentially_trained_model
             elif self.learning_approach in ['fedavg', 'seq-with-final-agg', 'seqavg']:
                 model_to_evaluate = self.build_model_from_weights(self.aggregate_model_weights())
-            model_evaluation = model_to_evaluate.evaluate(
-                x_val, y_val, batch_size=constants.DEFAULT_BATCH_SIZE, verbose=0,
-            )
+
+
+            model_evaluation = self.collaborative_round_evaluation(model_to_evaluate, self.val_data)
```

Comment:
Same thing here : `val_evaluation `or `val_score`.
And the same applies in the all the other occurences of calls to `collaborative_round_evaluation()`